content/news/playtest-20150524.md

```diff
@@ -0,0 +1,57 @@
+  - Several enhanced classic missions have been added to the Tiberian Dawn mod.
+  - In the Red Alert and D2k mods you will now be notified upon encounter of enemy units.
+
+Unfortunately, an oversight in the way that we package our windows installer meant that the performance of our Windows release-20150429 builds was much worse than it should have been. We have fixed this for playtest-20150524, so Windows players should find the game runs about 30% faster!<br>
+Also general performance improvements have been integrated that affect all platforms. Kudos to [RoosterDragon](https://github.com/RoosterDragon) for his continued effort to optimize our source code.
+
+The refined system integration adds a lot of convenient features:
+
+  - The hardware cursors will now work more reliably on Microsoft Windows.
+  - On Mac the dialogs won't hijack finder and the dock icon won't get lost upon restarting.
+  - Linux AppStream compatible software centers will now show a nice preview.
+  - Double-clicking an `.orarep` file will now directly start playback of the replay.
+
+Our new in-game map editor took over a year of development time and it was worth it.
+
+  - It is truly cross-platform as it uses our game UI; finally a map editor on Mac OS X!
+  - No more rendering glitches. Almost WYSIWYG editing, because we re-use the game renderer.
+  - Support for all file formats, i.e. voxels, isometric terrain and Dune 2000 sprite actors.
+  - You can now keep an overview of installed custom maps and delete them in the same menu.
+
+Click `Extras` - `Map Editor` from the main menu in any mod. We will remove the legacy `OpenRA.Editor.exe` in the nearby future.
+
+Great news for tinkerers. You can now use voxels in your own mods and maps. See our updated [trait documentation](https://github.com/OpenRA/OpenRA/wiki/Traits-(playtest)).
+The [Lua API](https://github.com/OpenRA/OpenRA/wiki/Lua-API-(playtest)) for mission scripting has also been enhanced while we proceed to implement remakes of the original campaigns.
+
+More details and links to pull request can be found on our [changelog page](http://changelog.openra.net/).
+
+You can find the installers for a variety of supported operating systems in our [download section](/download/).
+
+Please report problems [on our bug tracker](http://bugs.openra.net). We plan to do a new stable release based upon these changes soon.
+
+<div style="text-align:center" markdown="1">
+
+![Red Alert map chooser](/images/news/20150524-ra-map-chooser.png)
+
+Keep track of your thirdparty maps from [OpenRA Resource Center](http://resource.openra.net/) and delete unwanted ones in the updated map chooser.
+
+![Tiberian Dawn mission Nod06a](/images/news/20150524-cnc-nod06a.png)
+
+One by one the results of the student project to import the original Tiberian Dawn missions is integrated. Thanks to the [THM](http://www.thm.de/site/en/) course instructors for their creative teaching methods.
```

Comment:
It would look a bit nicer if you added an explicit `<br>` between the two sentences.